programs/adapter-jupiter/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "adapter-jupiter"
+version = "0.1.0"
+description = "Created with Anchor"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "lib"]
+name = "adapter_jupiter"
+
+[features]
+no-entrypoint = []
+no-idl = []
+no-log-ix-name = []
+cpi = ["no-entrypoint"]
+default = []
+
+[profile.release]
+overflow-checks = true
+
+[dependencies]
+anchor-lang = "0.24.2"
```

Comment:
Upgrade to v0.26.0

Reply:
I tried to build with v0.26.0, however, all adapters should also update to v0.26.0 to proceed compile.
Shall I push all updates on Cargo.toml in this repo?

Reply:
yes, plz

Reply:
OKay.